### PR TITLE
Broken link audit pt.2 (August 2023)

### PR DESCRIPTION
**Pull request by bre-fitzgerald**

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/about.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Here are some great use cases to get you inspired!
 {: .reset-td-br-1 .reset-td-br-2}
 
 
-[1]: {{site.baseurl}}/user_guide/message_building_by_channel/email/creating_an_email_campaign/
+[1]: {{site.baseurl}}/user_guide/message_building_by_channel/email/html_editor/creating_an_email_campaign//
```
Comment (rachel-feinberg) [inline _docs/_user_guide/message_building_by_channel/email/about.md:39]:
```suggestion
[1]: {{site.baseurl}}/user_guide/message_building_by_channel/email/html_editor/creating_an_email_campaign/
```

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Just one minor edit

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

**Comment by lydia-xie** [comment]
@cla-bot check

File: _docs/_user_guide/message_building_by_channel/email/about.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/message_building_by_channel/email/about.md]: Thanks for catching this!!

**Pull request merged**